up-core-api/uprotocol/uprotocol_options.proto

```diff
@@ -70,6 +71,19 @@ extend google.protobuf.MessageOptions {
   optional uint32 base_topic_id = 51300;
   // Code Access Permission (CAP) Level for the topic
   uint32 topic_permission_level = 51301;
+  
+  // Delivery notification request for published messages
+  // Indication to uTransport-specific instance that a notification should be provided to the publisher of a message
+  //  on completion/termination (with success/failure)  
+  optional bool notify_publisher = 51302; 
+}
+
+// uPublish notification message, intended to deliver context-specific content known the publisher, and provided by the uTransport layer supporting the Notiication response
```

Comment:
This documentation does not make any sense. and uprotocol_options does not make sense for what you are trying to do.